wire-ios/Wire-iOS/Sources/UserInterface/DeviceView/DeviceDetailsViewActionsHandler.swift

```diff
@@ -21,6 +21,10 @@ import WireDataModel
 import WireSyncEngine
 
 final class DeviceDetailsViewActionsHandler: DeviceDetailsViewActions, ObservableObject {
+    private static let logger = Logger(
+        subsystem: "DeviceDetailsView Actions",
+        category: "DeviceDetailsView"
+    )
```

Comment:
I would prefer to stick with current loggers we have.
For instance, WireLogger and create one for E2EI - it doesn't have to be specific to this class

Reply:
done